scipy/stats/tests/test_stats.py

```diff
@@ -4056,7 +4147,7 @@ def test_ttest_ind_permutation_nanpolicy(self):
             assert_allclose(res.statistic[~mask], res2.statistic)
 
             # Propagate 1d
-            res = stats.ttest_ind(a.ravel(), b.ravel(), **options_p)
+            assert(np.isnan(res.pvalue))  # assert makes sure it's a scalar
```

Comment:
You commit must not change lines in other functions.

Seems to be a mistake in editing.